site/content/user_guide.md

```diff
@@ -4,13 +4,13 @@ While you are welcome to provide your own organization, typically a Cobra-based
 application will follow the following organizational structure:
 
 ```console
-  ▾ appName/
-    ▾ cmd/
-        add.go
-        your.go
-        commands.go
-        here.go
-      main.go
+▾ appName/
+  ▾ cmd/
+      add.go
+      your.go
+      commands.go
+      here.go
+  main.go
```

Comment:
Not sure if main.go is indented correctly. We can replace it with tree output which is much more clear to users and also used later below.

```console
appName
├── cmd
│   ├── add.go
│   ├── your.go
│   ├── commands.go
│   └── here.go
└── main.go
```

Reply:
Yeah, feel free to change this if you prefer.

Reply:
In a follow-up 😄